### DISCUSSION TITLE
Seq. Unable to start service

### DISCUSSION THREAD

**Discussion by rsiladii-Ameria** [datalust/seq-tickets#2047] [Help]
Seq service stop after a few seconds. Main error ERR] Cache warmup failed; the system will continue managing cache resources, however this problem indicates critical storage issues and should be investigated
Please HELP!

**Comment by KodrAus** [comment]
Hi @rsiladii-Ameria :wave: Seq should produce some more diagnostic logs under `C:\ProgramData\Seq\Logs`. If you send those through to our support address at `support@datalust.co` we can take a look for you.
